browser/main/index.js

```diff
@@ -27,7 +27,10 @@ document.addEventListener('click', function (e) {
   const className = e.target.className
   if (!className && typeof (className) !== 'string') return
   const isInfoButton = className.includes('infoButton')
-  const isInfoPanel = e.target.offsetParent.className.includes('infoPanel')
+  let isInfoPanel = false
+  if (e.target.offsetParent !== null) {
+    isInfoPanel = e.target.offsetParent.className.includes('infoPanel')
+  }
```

Comment:
Cool! However, I prefer this way

```js
  const offsetParent = e.target.offsetParent
  const isInfoPanel = offsetParent !== null
    ? offsetParent.className.includes('infoPanel')
    : false
```

How about this?

Reply:
Sure. I will update that in a bit. The only reason I did it the way I did is for readability. Ternary statement it is!